discord/ext/ipc/client.py

```diff
@@ -21,13 +21,15 @@
 
 class Client:
     """Handles webserver side requests to the bot process.
```

Comment:
```suggestion
    """
    Handles webserver side requests to the bot process.
```

Reply:
This fits the description for a short summary, according to the standards being followed for documentation. But can do

Reply:
I'm sure it does, this is a personal preference.